charts/kong/templates/_helpers.tpl

```diff
@@ -166,7 +166,9 @@ spec:
   - name: kong-{{ .serviceName }}
     port: {{ .http.servicePort }}
     targetPort: {{ .http.containerPort }}
-    appProtocol: http
+  {{- if .http.appProtocol }}
+    appProtocol: {{ .http.appProtocol }}
+  {{- end }}
```

Comment:
Why would we set the `appProtocol` to `true`?

Reply:
you are right
fixed typo